Thinktecture.Relay.Server/App.config

```diff
@@ -19,6 +19,10 @@
     <add key="serilog:using:Seq" value="Serilog.Sinks.Seq" />
     <add key="serilog:write-to:Seq.serverUrl" value="http://localhost:5341/" />
 
+    <add key="RabbitMqClusterHosts" value="localhost;localhost:5673;localhost:5674;bullshit:port;some:serious:shit" />
```

Comment:
We should not commit such a configuration.

Reply:
Indeed ;)